.vitepress/config.mts

```diff
@@ -1,132 +1,135 @@
-import { defineConfig } from 'vitepress';
+import { defineConfig } from "vitepress";
 
 export default defineConfig({
-  srcDir: 'docs',
+  srcDir: "docs",
   srcExclude: [],
-  outDir: 'build',
-  base: '/',
-  title: 'The JavaScript Oxidation Compiler',
-  titleTemplate: ':title | The JavaScript Oxidation Compiler',
+  outDir: "build",
+  base: "/",
+  title: "The JavaScript Oxidation Compiler",
+  titleTemplate: ":title | The JavaScript Oxidation Compiler",
   description:
-    'A collection of high-performance JavaScript tools written in Rust',
+    "A collection of high-performance JavaScript tools written in Rust",
   head: [
     [
-      'link',
+      "link",
       {
-        rel: 'icon',
-        href: 'https://raw.githubusercontent.com/oxc-project/oxc-assets/main/logo-round.png',
+        rel: "icon",
+        href: "https://raw.githubusercontent.com/oxc-project/oxc-assets/main/logo-round.png",
       },
     ],
   ],
   lastUpdated: true,
   themeConfig: {
-    siteTitle: 'Oxc',
-    logo: 'https://raw.githubusercontent.com/oxc-project/oxc-assets/main/logo-round.png',
-    logoLink: '/',
+    siteTitle: "OXC",
+    logo: "https://raw.githubusercontent.com/oxc-project/oxc-assets/main/logo-round.png",
+    logoLink: "/",
     search: {
-      provider: 'local',
+      provider: "local",
     },
     nav: [
-      { text: 'Home', link: '/' },
-      { text: 'Getting Started', link: '/docs/usage/linter' },
-      { text: 'Contribute', link: '/docs/contribute/intro' },
-      { text: 'Learn', link: '/docs/learn/ecosystem' },
-      { text: 'Blog', link: '/blog/2022-02-10-js-tooling-research' },
+      { text: "Home", link: "/" },
+      { text: "Getting Started", link: "/docs/usage/linter" },
+      { text: "Contribute", link: "/docs/contribute/intro" },
+      { text: "Learn", link: "/docs/learn/ecosystem" },
+      { text: "Blog", link: "/blog/2022-02-10-js-tooling-research" },
       {
-        text: 'Playground',
-        target: '_blank',
-        link: 'https://oxc-project.github.io/oxc/playground/',
+        text: "Playground",
+        target: "_blank",
+        link: "https://oxc-project.github.io/oxc/playground/",
       },
     ],
     socialLinks: [
-      { icon: 'twitter', link: 'https://x.com/boshen_c' },
-      { icon: 'discord', link: 'https://discord.gg/9uXCAwqQZW' },
-      { icon: 'github', link: 'https://github.com/oxc-project' },
+      { icon: "twitter", link: "https://x.com/boshen_c" },
+      { icon: "discord", link: "https://discord.gg/9uXCAwqQZW" },
+      { icon: "github", link: "https://github.com/oxc-project" },
     ],
     editLink: {
-      pattern: 'https://github.com/oxc-project/oxc/edit/main/src/:path',
+      pattern: "https://github.com/oxc-project/oxc/edit/main/src/:path",
     },
     lastUpdated: {
       formatOptions: {
-        dateStyle: 'full',
+        dateStyle: "full",
       },
     },
+    footer: {
+      copyright: "© 2023 OXC Project",
+    },
```

Comment:
@Boshen 

Would you tell the initial year of this project?

Reply:
2023 is the year.